src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -25,6 +25,9 @@
     -->
     <_GetChildProjectCopyToPublishDirectoryItems>false</_GetChildProjectCopyToPublishDirectoryItems>
     <UseMonoRuntime Condition=" '$(UseMonoRuntime)' == '' ">true</UseMonoRuntime>
+
+    <!-- Use $(AndroidMinimumSupportedApiLevel) for $(SupportedOSPlatformVersion) if unset -->
+    <SupportedOSPlatformVersion Condition=" '$(SupportedOSPlatformVersion)' == '' ">$(AndroidMinimumSupportedApiLevel)</SupportedOSPlatformVersion>
```

Comment:
I don't understand how this *works*: I don't see any instances of `<AndroidMinimumSupportedApiLevel` within this PR, and existing matches within xamarin-android@fb107693 are for an *item group*, not a property.  What sets `$(AndroidMinimumSupportedApiLevel)` *on the end-user machine*?

(…or is this a "no-op" because `$(SupportedOSPlatformVersion)` is *always* set, and thus this is moot?)

Reply:
This property is set in the generated `Microsoft.Android.Sdk.SupportedPlatforms.props` file, which is imported before this file. The property is set using the value of `$(AndroidMinimumDotNetApiLevel)` here:
https://github.com/xamarin/xamarin-android/blob/6eb11f159c692563200ffb88cddc7b69f761cace/build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/GenerateSupportedPlatforms.cs#L67

Reply:
Thanks! 👍